Flutter/pdf/working-with-pdf-templates.md

```diff
@@ -43,6 +43,37 @@ document.dispose();
 
 {% endhighlight %}
 
+## Creating templates from existing PDF document
+
+Essential PDF supports creating the templates using [`CreateTemplate`](https://pub.dev/documentation/syncfusion_flutter_pdf/latest/pdf/PdfPage/createTemplate.html) method from an existing PDF document page and draw it in on a new PDF document.
```

Comment:
Essential PDF supports template creation using the "CreateTemplate" method, enabling users to extract content from an existing PDF page and seamlessly incorporate it into a new PDF document.